haskell-src-exts.cabal

```diff
@@ -88,6 +88,7 @@ Test-Suite test
                        tasty >= 0.3,
                        tasty-smallcheck,
                        tasty-golden >= 2.2.2,
+                       tasty-hunit,
```

Comment:
I agree that it is not good idea to add dependency.

I did it because AFAIK there is no library for unit tests in this project. I though about using property based testing, but after your suggestion that that could not be good idea, decided to write unit tests instead. 
But probably I should discuss this before preparing this PR.